File: 백준/Bronze/6376. e 계산/e 계산.cc
#include <cstdio>
#include <iostream>

#define endl "\n";

using namespace std;


int main() {
	cout << "n e" << endl;
	cout << "- -----------" << endl;
	cout << "0 1\n";
	cout << "1 2\n";
	cout << "2 2.5\n";

	double factorial = 2;

	double i = 2;

	double result = 2.5;
	
	while(factorial < 9) {
		factorial++;
		i *= factorial;

		result += 1 / i;

		printf("%.0f %.9f\n", factorial, result);
	}
}

File: 백준/Bronze/6376. e 계산/e 계산.py
import math
import sys

sys.setrecursionlimit(10 ** 6)
input = lambda: sys.stdin.readline().rstrip()


def solve(n):
    res = 1
    if n == 0:
        return 1

    for i in range(1, n + 1):
        res *= i

    return res


result = 0
print('n e')
print('- -----------')

for i in range(10):
    result += 1 / math.factorial(i)
    if result > 2.5:
        print(f'{i} {result:0.9f}')
    elif result == 1 or result == 2:
        print(f'{i} {result:0.0f}')
    elif result == 2.5:
        print(f'{i} {result:0.1f}')